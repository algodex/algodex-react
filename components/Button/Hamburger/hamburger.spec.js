import React from 'react'
import renderer from 'react-test-renderer'
import MenuButton from './index'

it('MenuButton: default', () => {
  const component = renderer.create(<MenuButton />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
