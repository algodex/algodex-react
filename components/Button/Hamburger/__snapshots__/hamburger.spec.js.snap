// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MenuButton: default 1`] = `
<svg
  className="css-145sz59-StyledMotionSvg e13rx5dl0"
  height={15}
  overflow="visible"
  preserveAspectRatio="none"
  style={Object {}}
  viewBox="0 0 6.4 4"
  width={24}
>
  <line
    stroke="#F8FAFC"
    strokeWidth={2}
    style={Object {}}
    vectorEffect="non-scaling-stroke"
    x1="0"
    x2={6.4}
    y1="0"
    y2="0"
  />
  <line
    opacity={1}
    stroke="#F8FAFC"
    strokeWidth={2}
    style={Object {}}
    vectorEffect="non-scaling-stroke"
    x1="0"
    x2={6.4}
    y1="2"
    y2="2"
  />
  <line
    stroke="#F8FAFC"
    strokeWidth={2}
    style={Object {}}
    vectorEffect="non-scaling-stroke"
    x1="0"
    x2={6.4}
    y1="4"
    y2="4"
  />
</svg>
`;
